extensions/sql-migration/src/api/utils.ts

```diff
@@ -230,6 +230,14 @@ export function get12HourTime(date: Date | undefined): string {
 	return (date ? date : new Date()).toLocaleTimeString([], localeTimeStringOptions);
 }
 
+export function displayDialogErrorMessage(dialog: window.Dialog, text: string, error: Error): void {
+	dialog.message = {
```

Comment:
dialog message doesn't need to handle localization?

Reply:
I noticed other places use Localize('','') but here not.

Reply:
this is a function that is taking parameters that use localized strings.